test/support/BaseSpec.scala

```diff
@@ -51,10 +55,21 @@ class BaseSpec extends PlaySpec with GuiceOneAppPerSuite with Injecting with Bef
   val validSessionToken: String = Session.generateToken("1234")
   val invalidSessionToken: String = Session.generateToken("invalid-user")
 
+  private lazy val testTimeProvider: TimeProvider = () => LocalDateTime.of(2023, Month.FEBRUARY, 28, 1, 1, 1)
+  private lazy val testConfig: Config = Config(
+    broker = HostBrokerEndpoint("not.a.real.host", 0),
```

Comment:
I wanted to make it explicit that this isn't meant to connect to anything, and make it an easy string to search for (as opposed to "localhost")

Reply:
Yes. And BaseSpec will soon be slimmed down significantly as part of making these actual unit tests.